aeron-cluster/src/main/java/io/aeron/cluster/Election.java

```diff
@@ -1328,6 +1328,31 @@ private void ensureRecordingLogCoherent(
         final long nowNs)
     {
         final long recordingId = consensusModuleAgent.logRecordingId();
+        final long initialTermBaseLogPosition = this.initialTermBaseLogPosition;
```

Comment:
Why create local vars here?

Reply:
That was part of a extract method refactor and I forgot to inline the fields back into the method call.